dragonflybsd/Platform.c

```diff
@@ -169,15 +169,17 @@ double Platform_setCPUValues(Meter* this, int cpu) {
       v[CPU_METER_KERNEL]  = cpuData->systemPercent;
       v[CPU_METER_IRQ]     = cpuData->irqPercent;
       this->curItems = 4;
-      percent = v[0]+v[1]+v[2]+v[3];
+      percent = v[0] + v[1] + v[2] + v[3];
    } else {
       v[2] = cpuData->systemAllPercent;
       this->curItems = 3;
-      percent = v[0]+v[1]+v[2];
+      percent = v[0] + v[1] + v[2];
    }
 
    percent = CLAMP(percent, 0.0, 100.0);
-   if (isnan(percent)) percent = 0.0;
+   if (isnan(percent)) {
```

Comment:
strictly speaking this is dead code as `CLAMP(NAN, $min, $max) == $min`

Reply:
I'd prefer to still have it explicit, as the implementation of CLAMP might change and you don't want to rely on its implementation. Thus shortened to: `percent = isnan(percent) ? 0.0 : CLAMP(percent, 0.0, 100.0);`